app/src/main/java/com/eveningoutpost/dexdrip/NSEmulatorReceiver.java

```diff
@@ -110,6 +110,7 @@ public void run() {
                                                         // sanity checking???
                                                         // fake up some extra data
                                                         faux_bgr.put("raw_data", json_object.getDouble("sgv"));
+                                                        faux_bgr.put("age_adjusted_raw_value", json_object.getDouble("sgv"));
```

Comment:
I'm not exactly sure what this will do but it is okay for completeness. I didn't think that field was used if `hasLibre()` is true.

Reply:
This was needed when I have started looking at applying calibrations on the oop algorithm as well.
But this will wait for the next pr.